test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -173,13 +173,13 @@
                             "R4",
                             "Num",
                             "R8",
-                            "TX",
                             "Text",
+                            "TX",
```

Comment:
This is also due to the enum inconsistency  but I couldn't find a good spot to make this more consistent. 

Just updating the baseline for this instance.